messages/config-command.md

```diff
@@ -0,0 +1,50 @@
+# command.summary
+
+See the current state of the config for Code Analyzer.
+
+# command.description
+
+See a YAML-formatted config object describing Code Analyzer's default configuration. Or see a final config created from
+the resolution of an existing config, rule selectors, and a workspace. Optionally, write the results to a file.
+
+# command.examples
+
+- Display the default configuration for Code Analyzer:
+
+     <%= config.bin %> <%= command.id %>
+
+- Display the default configuration for Code Analyzer and write it to a file:
+
+     <%= config.bin %> <%= command.id %> --output-file code-analyzer.yml
+
+# flags.workspace.summary
+
+*DRAFT*
+
+# flags.workspace.description
+
+*DRAFT*
+
+# flags.rule-selector.summary
+
+*DRAFT*
+
+# flags.rule-selector.description
+
+*DRAFT*
+
+# flags.config-file.summary
+
+*DRAFT*
+
+# flags.config-file.description
+
+*DRAFT*
+
+# flags.output-file.summary
+
+Output file that contains the final config. Will always be formatted as YAML.
```

Comment:
My recommendation:
```
Output file to write the configuration state to. The format will written in YAML format.
```

Note: we should restrict what the user can put as the file extension to only be ".yml" or ".yaml". This way, in the future if users for some odd reason what to write to json... (which we can easily do - with the exception of not being able to give them comments in the file) then we can do so based on the file extension. If we currently allow them to write yaml to other file extensions (including .json) then this will block us from being flexible in the future without introducing breaking changes.

Reply:
"Format will be written in YAML format" is redundant. How about "Format will be YAML"?

Reply:
Or "File will be written in YAML format?"